modelina-website/src/pages/api/functions/JavaGenerator.ts

```diff
@@ -16,6 +17,55 @@ export async function getJavaModels(
   };
   applyGeneralOptions(generatorOptions, options, javaDefaultEnumKeyConstraints, javaDefaultPropertyKeyConstraints, javaDefaultModelNameConstraints);
 
+  if (generatorOptions.javaIncludeJackson) {
+    options.presets?.push(JAVA_JACKSON_PRESET)
+  }
+
+  if (generatorOptions.javaIncludeMarshaling) {
+    options.presets?.push(JAVA_COMMON_PRESET)
+  }
+
+  if (generatorOptions.javaArrayType) {
+    options.collectionType = generatorOptions.javaArrayType;
+  }
+
+  if (generatorOptions.javaOverwriteHashcode) {
+    options.presets?.push(
+      {
+        preset: JAVA_COMMON_PRESET,
+        options: {
+          hashCode: generatorOptions.javaOverwriteHashcode
+        }
+      }
+    )
+  }
+
+  if(generatorOptions.javaOverwriteEqual){
+    options.presets?.push({
+      preset: JAVA_COMMON_PRESET,
+      options: {
+        equal: generatorOptions.javaOverwriteEqual
+      }
+    })
+  }
+
+  if(generatorOptions.javaOverwriteToString){
+    options.presets?.push({
+      preset: JAVA_COMMON_PRESET,
+      options: {
+        classToString: generatorOptions.javaOverwriteToString
+      }
+    })
+  }
```

Comment:
These options and `javaIncludeMarshaling` does not work together, and that's because you have to set all the remaining options to false, and only one to true.

i.e. for this one it should be:

```
    {
      preset: JAVA_COMMON_PRESET,
      options: {
        equal: false,
        hashCode: false,
        classToString: true
      }
    }

```